sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -40,7 +40,7 @@ export class CASService {
     };
     const data = stringify(body);
     try {
-      return await this.httpService.axiosRef.post(url, data, config);
+      return (await this.httpService.axiosRef.post(url, data, config)).data;
```

Comment:
For these cases, we create a const most of the time. I would recommend doing the same here.
```ts
const response = await this.httpService.axiosRef.post(url, data, config);
return response .data;
```